src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorOldFormat = fromFormat[3];
+  const separatorNewFormat = toFormat[3];
+
+  const oldFormat = date.split(separatorOldFormat);
+  const newFormat = [];
+
+  for (let i = 0; i < oldFormat.length; i++) {
+    if (fromFormat[i].includes('Y') && toFormat.indexOf(fromFormat[i]) < 0) {
```

Comment:
For such long condition better to create a separate variable to describe what you expect
```suggestion
		const isDifferentYearFormat = fromFormat[i].includes('Y') && toFormat.indexOf(fromFormat[i]) < 0;

    if (isDifferentYearFormat) {
```